build_projects/dotnet-host-build/MsiTargets.cs

```diff
@@ -162,6 +162,7 @@ public static BuildTargetResult GenerateDotnetHostFxrMsi(BuildTargetContext c)
             var inputDir = c.BuildContext.Get<string>("HostFxrPublishRoot");
             var wixObjRoot = Path.Combine(Dirs.Output, "obj", "wix", "hostfxr");
             var hostFxrBrandName = $"'{Monikers.GetHostFxrBrandName(c)}'";
+			var upgradeCode = Utils.GenerateGuidFromName(HostFxrMsi).ToString().ToUpper();
```

Comment:
One of the acquisition requirement for Hostfxr is to support side by side for build-to-build . This can be achieved by generating unique UpgradeCode GUID that is passed to MSI authoring.